proto/storage/image.proto

```diff
@@ -118,17 +121,17 @@ message EmbeddedImageScanComponent {
     string                         name        = 1 [(gogoproto.moretags) = 'search:"Component,store"'];
     string                         version     = 2 [(gogoproto.moretags) = 'search:"Component Version,store"'];
     License                        license     = 3;
-    repeated EmbeddedVulnerability vulns       = 4;
+    repeated EmbeddedVulnerability vulns       = 4 [(gogoproto.moretags) = 'hash:"set"'];
     oneof has_layer_index {
         int32 layer_index                      = 5;
     }
-    int64                          priority    = 6;
+    int64                          priority    = 6 [(gogoproto.moretags) = 'hash:"ignore"'];
     SourceType                     source      = 7;
     string                         location    = 8;
     oneof set_top_cvss {
         float                      top_cvss    = 9;
     }
-    float                          risk_score  = 10 [(gogoproto.moretags) = 'search:"Component Risk Score,hidden"'];
+    float                          risk_score  = 10 [(gogoproto.moretags) = 'search:"Component Risk Score,hidden" hash:"ignore"'];
```

Comment:
perhaps contentious, need to look more closely at this

Reply:
I think we should not ignore the changes to the risk score in deduper, else we may have incorrect risk in db.

(please address this comment before merging)

Reply:
Yeah will do. I saw something for image risk score that looked like it was related to floating point math so will check to see if components are affected in the same way